CHANGELOG.md

```diff
@@ -32,6 +38,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   incorrect candidate sets when reused in different optimization contexts
 - `qNIPV` not working with single `MIN` targets
 
+### Removed
+- `botorch_function_wrapper` utility for creating lookup callables
```

Comment:
@Scienfitz, FYI since you already reviewed: I just also dropped this helper since it's no longer needed. You can simply resolve the thread once you saw it